lib/utils/temp.txt

```diff
@@ -0,0 +1,214 @@
+1	Log10 [Unity]
+1	MillionPerSpecimen [Unity]
+1	PerTotalCount [Unity]
+1	Thousand [Unity]
+1	ThousandRedBloodCells [Unity]
+1	OneHundredThousand [Unity]
+1	Million [Unity]
+1	TenToEighth [Unity]
+1	Percent [General Fraction Unit]
```

Comment:
What's this file?

Reply:
Apologies, this was a work file I had that I forgot to delete.

Reply:
6202929c8d20b3ad709e9c1fbfefb7d2cf01b11d